db/migrations/versions/~2023_10_02_1104-a0a14c197848_.py

```diff
@@ -0,0 +1,35 @@
+"""empty message
+
+Revision ID: a0a14c197848
+Revises: c2a534a4b74a
+Create Date: 2023-10-02 11:04:03.810662
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "a0a14c197848"
+down_revision = "c2a534a4b74a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("round", schema=None) as batch_op:
+        batch_op.drop_constraint("uq_round_short_name", type_="unique")
+        batch_op.create_unique_constraint(
+            batch_op.f("uq_round_fund_id"), ["fund_id", "short_name"]
+        )
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("round", schema=None) as batch_op:
+        batch_op.drop_constraint(batch_op.f("uq_round_fund_id"), type_="unique")
+        batch_op.create_unique_constraint("uq_round_short_name", ["short_name"])
```

Comment:
Nice, not sure how `batch_op.create_unique_constraint("uq_round_short_name", ["short_name"])` will go after we already have duplicate short names in place following the new composite key?

Reply:
> Nice, not sure how `batch_op.create_unique_constraint("uq_round_short_name", ["short_name"])` will go after we already have duplicate short names in place following the new composite key?

Confused what you mean by this?

Reply:
Like.  After your migration is in, 2 rounds now exist with the same short name, but different fund ids.  However if we downgrade the database, and re-create this unique constraint, it might complain or fail, if that makes sense